code/modules/hydroponics/grown.dm

```diff
@@ -65,16 +67,26 @@
 	make_dryable()
 
 	// Go through all traits in their genes and call on_new_plant from them.
+	// TODO: we need a priority queue for traits,some of them need to be called first
 	for(var/datum/plant_gene/trait/trait in seed.genes)
 		trait.on_new_plant(src, loc)
 
+	// needs to be run after traits are called because some of them alter max_volume and volume_rate
+	// since traits do not know in which order they were run we need to do it here
```

Comment:
```suggestion
	//Needs to be run after traits are called because some of them alter max_volume and volume_rate
	//Since traits do not know in which order they were run we need to do it here
```